src/pini.cpp

```diff
@@ -65,7 +81,20 @@ std::int32_t pini::get_int32(std::string const& key, std::uint32_t def) const {
 
 std::string_view pini::get_string(std::string const& key) const {
 	auto it = key_value_pairs.find(key);
-	if (it == key_value_pairs.end()) { return {}; }
+	if (it == key_value_pairs.end()) {
+		if (pn::pini::on_msg_t) {
```

Comment:
Are you ok with repeating this pattern every time you need to log stuff? There are ways to make it more succinct.